packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2155,24 +2095,21 @@ function commitRootImpl(root, renderPriorityLevel) {
 
 export function flushPassiveEffects(): boolean {
   // Returns whether passive effects were flushed.
+  // TODO: This depends on the scheduler priority, which is no longer updated.
```

Comment:
Need to look into this before merging. I think we need to switch this over to using the update lane priority.

Reply:
If it's Offscreen, it should be Offscreen
If it's Idle, it should be Idle
Everything else should be Default

Reply:
Will follow up with this. I think I kept the same behavior, but using the update lane instead of the scheduler lane.